support/tech-resources/deep-links-self-serve.md

```diff
@@ -5,152 +5,230 @@ description: "A guide to including iOS universal links and Android App Links in
+
+### HTTPS secure links
+
+It is good practice to use secure (HTTPS) tracking domains, and it's essential if your website has an [HSTS](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) policy. We have articles describing the setup of this for:
+
+-  [Various CDNs](https://www.sparkpost.com/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost/)
+- Your own [web server reverse proxy such as NGINX](https://www.sparkpost.com/docs/tech-resources/using-proxy-https-tracking-domain/)
+
+In both cases, your CDN (or proxy) redirects the specific tracking subdomain(s) to the SparkPost click tracking service, while your web content is served in the usual way.
+
+### How click tracking works
+
+It might be useful to understand how the SparkPost click tracking feature works with HTML links. Other email-service-provider click tracking services work in a similar way.
+
+> When SparkPost encounters an anchor tag in an HTML email, it will replace the `href` attribute with a new URL pointing to the SparkPost click tracking service. When your recipient clicks that link, the SparkPost service receives the request (via CDN for HTTPS links), records the click, and redirects the recipient to your original URL.
+
+![](media/deep-links-self-serve/deep-links-click-tracking-simple.png)
+*Normal click tracking (without deep links)  icons by [The Noun Project](https://thenounproject.com/)*
+
+Deep links supersede this flow; after checking the spec file, the  device passes the request directly to your app instead. This means your app will receive the tracked link via an API call and can show the user an appropriate part of your app.
+
+SparkPost won't receive the request and won't register the click in your analytics, unless your app code makes it happen.
+
+A workaround is to disable tracking for deep links. This may be appropriate if you already have analytics for measuring in-app activity.
+
+The preferred solution is to set up Sparkpost click tracking and include code in your app to follow the link. We'll review each of these.
+
+### Workaround: Disabling Tracking of deep links
+
+If you disable SparkPost's click tracking for the mobile deep links in your email, they work just fine with your mobile app with no further effort. This works because SparkPost does not alter untracked links in your email.
+
+The "path" in your Apple spec file needs to match your iOS app "associated domains".
+
+If you're using the SparkPost REST API, you can use the `data-msys-clicktrack` attribute to [disable click tracking for a single link in your email](https://developers.sparkpost.com/api/template-language/#header-per-link-disabling-of-click-tracking):
 
 ```html
-<a href="http://my.universal-link.example.com" data-msys-sublink="open-in-app">Open in app</a>
+<a data-msys-clicktrack="0" href="http://my.universallink.example.com/path/">Open in app</a>
```

Comment:
I think we should mention how to do this for text parts too. From https://developers.sparkpost.com/api/template-language/#header-custom-link-attributes

`http://www.example.com[[data-msys-clicktrack="0"]]`

Reply:
OK, will do.
You can also enable deep-linking from a text email using [[data-msys-sublink=\"open-in-app\"]]. Tested on Apple and works. Will add that too.